config/betterquesting/DefaultQuests.json

```diff
@@ -1879,7 +1879,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Crushed Ores?",
-          "desc:8": "Explaining the whole ore processing is out of the scope of this introduction, so we will keep it simple for now. More details in LV.\n\nMacerating is still an ore doubler like in other mods. But there is a twist, it does not give dusts directly! Crushed Ores are meant for more advanced ore processing. One Crushed Ore will convert to one dust.\n\n\nTo get the an Impure Dust, hammer the Crushed Ore. This is done either by using a Hammer tool in the crafting table, or using the Steam Forge Hammer.\n\nThe Impure Dust can be smelted in the Furnace to get the Ingot, however mixed ores (Limonite, Chalcopyrite) will only give nuggets over ingots.\n\nThe Impure Dust can also be thrown in a vanilla Cauldron filled with water to clean it and turn it into a regular dust. Throw them in full stacks to consume less water per single dust.\n\n\n§lNote: Macerating, hammering, and then smelting (sometimes into nuggets) is really time consuming in the steam age! Resource efficiency can be ignored for now. Nobody will blame you for smelting ores directly into ingots."
+          "desc:8": "Explaining the whole ore processing system is out of the scope of this introduction, so we will keep it simple for now. More details in LV.\n\nMacerating is still an ore doubler like in other mods. But there is a twist, it does not give dusts directly! Crushed Ores are meant for more advanced ore processing. One Crushed Ore will convert to one dust.\n\n\nTo get the an Impure Dust, hammer the Crushed Ore. This is done either by using a Hammer tool in the crafting table, or using the Steam Forge Hammer.\n\nThe Impure Dust can be smelted in the Furnace to get the Ingot, however mixed ores (Limonite, Chalcopyrite) will only give nuggets over ingots.\n\nThe Impure Dust can also be thrown in a vanilla Cauldron filled with water to clean it and turn it into a regular dust. Throw them in full stacks to consume less water per single dust.\n\n\n§lNote: Macerating, hammering, and then smelting (sometimes into nuggets) is really time consuming in the steam age! Resource efficiency can be ignored for now. Nobody will blame you for smelting ores directly into ingots."
```

Comment:
This quest comes after the Macerator so I don't think it's worth mentioning. Maybe in an earlier quest though? It might already be there, not sure. Not feeling like checking right now :stuck_out_tongue: